docs/using-connect/integration.mdx

```diff
@@ -0,0 +1,374 @@
+---
+description: Integrating Connect with Blockchain Applications
+title: Integration
+icon: swap-arrows
+---
+
+<Info>
+    Connect is business-licensed software under the Business Source License (BSL). The source code is viewable; however, please reach out to us if you are interested in integrating. We are limiting the number of partnerships we engage with in 2024. We apologize in advance if we reach capacity and are unable to accommodate new integrations.
+</Info>
+
+
+
+This document will guide you through integrating Connect in your application.
+
+## Requirements
+- Go 1.22+
+- Cosmos SDK v0.50+
+
+
```

Comment:
spacing in this page is off

Reply:
is there a .mdx formatting tool we can use?

Reply:
https://github.com/markdownlint/markdownlint

Reply:
fixed, will look into that tool. [spacing](https://github.com/skip-mev/slinky/pull/618/commits/890b8ee67073a05d985a9a8053fe8850735b1c07)